CONTRIBUTING.md

```diff
@@ -43,6 +43,49 @@ Use these commands whenever you want to work on one of these packages. Ideally,
 | `pnpm header:start`               | starts the stencil compiler for the header                                                   |
 | `pnpm icons:start`                | starts the http server for debugging downloaded icons                                        |
 
+#### Definition of Done for a Story
+
+- Component docs
```

Comment:
Could you convert all the top-level items (Component docs, Controls, Testing, etc.) to a h4 (e.g. #### Component docs)?